chroot-bin/croutonnotify

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh -e
+# Copyright (c) 2015 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+USAGE="${0##*/} -j
+${0##*/} -t title [-m message] [-d display] [-i id]
+Raises a notification in Chromium OS (requires crouton extension).
+
+When -j is specified, reads JSON data from stdin, in the format required
+by chrome.notifications API, with 2 additional fields, \"crouton_id\" and
+\"crouton_display\", corresponding respectively to the notification id, and the
+display to switch to when the notification is clicked (croutoncycle parameter).
+
+Otherwise, constructs a \"basic\" notification with the requested fields.
+
+Options:
+    -j       Read JSON data from stdin (see example below)
+    -t       Title to display (chrome.notifications field: \"title\")
+    -m       Message to display (chrome.notifications field: \"message\")
+    -d       Display to switch to when the notification is clicked (croutoncycle
```

Comment:
Won't the vast majority of -d parameters just be "$DISPLAY"?
How about making -d parameterless and just use the $DISPLAY variable, which really just works the same as notify-send.